crates/sncast/src/starknet_commands/account/create.rs

```diff
@@ -52,6 +55,7 @@ pub async fn create(
     let class_hash = class_hash.unwrap_or_else(|| {
         FieldElement::from_hex_be(OZ_CLASS_HASH).expect("Failed to parse OZ class hash")
     });
+    check_class_hash_exists(provider, class_hash).await?;
```

Comment:
Can we make the same check in the `account add` command?

Reply:
As I see, we don't perform any validations when doing `account add`
Introducing class hash validation we should also validate account address if the `--deployed` flag is passed
WDYT?

Reply:
Agree, it makes sense

Reply:
Added in https://github.com/foundry-rs/starknet-foundry/pull/1774/commits/ea8356603bcee52c75eb5dd7c77bcf2ced8a93be

And created an issue for account address verification https://github.com/foundry-rs/starknet-foundry/issues/1780